pkg/kubecost/allocation.go

```diff
@@ -63,6 +63,8 @@ type Allocation struct {
 	GPUHours                   float64               `json:"gpuHours"`
 	GPUCost                    float64               `json:"gpuCost"`
 	GPUCostAdjustment          float64               `json:"gpuCostAdjustment"`
+	NetworkTransferBytes       float64               `json:"networkTransferBytes"`
+	NetworkReceiveBytes        float64               `json:"networkReceiveBytes"`
```

Comment:
Should we also add these to the `MarshalJSON`, or do you want to keep them out of the JSON payload for now?

Reply:
Great Googly Moogly

Reply:
> Any notes on how this was tested?

I had logging after the query that seemed to indicate the data coming through was correct, but after adding JSON encoding, I'm seeing 0s for both on the allocation request for 7d, so it seems I need to find a better way to end-to-end test.

Suggestions other than observing on the JSON result?

Reply:
Added that, the fields to the equality check, and also to `add()`